src/Redux/APIFunctions/categories.js

```diff
@@ -0,0 +1,54 @@
+import { createApi, fetchBaseQuery } from '@reduxjs/toolkit/query/react';
+
+export const categoriesApi = createApi({
+  reducerPath: 'categories',
+  baseQuery: fetchBaseQuery({ baseUrl: 'https://stark-badlands-38572.herokuapp.com/api/v1' }),
+  tagTypes: ['Cats'],
+  endpoints: (builder) => ({
+    getCategories: builder.query({
+      query: () => '/categories',
+      // transformResponse: (res) => res.sort((a, b) => b.id - a.id),
```

Comment:
- [ ] Please, kindly remove this old piece of commented code, to improve the cleanliness of the repo 💪